build-analyze.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Configuration for SonarCloud
+SONAR_HOST_URL=https://sonarcloud.io
+#SONAR_TOKEN= # access token from SonarCloud projet creation page -Dsonar.login=XXXX: define in the environement through the CI
+export SONAR_SCANNER_VERSION="4.6.1.2450" # Find the latest version in the "Windows" link on this page:
+                                          # https://sonarcloud.io/documentation/analysis/scan/sonarscanner/
```

Comment:
This link is broken

Reply:
I copy-pasted it from another example.
It seems like it is broken in all the SC examples... We missed it when we did the changes due to the move of SC documentation. 
I'll do the change in all the relevant repos

Reply:
I have a half-automated way to perform find&replace in all our example repos. As we will have to change all of them to add links for comp-DB, coverage, and otherci, I propose I just put
```
https://sonarcloud.io/documentation/analysis/scan/sonarscanner/ https://docs.sonarcloud.io/advanced-setup/ci-based-analysis/sonarscanner-cli/
```
into the list of changes, and do all of them in a batch

Reply:
Seems good!